vendor/openvino/openvino.BUILD

```diff
@@ -0,0 +1,151 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # BSD/MIT-like license
+
+load("@com_intel_plaidml//bzl:template.bzl", "template_rule")
+
+cc_binary(
+    name = "benchmark_app",
+    srcs = glob([
+        "inference-engine/samples/benchmark_app/**/*.hpp",
+        "inference-engine/samples/benchmark_app/**/*.cpp",
+    ]),
+    data = [":plugins"],
+    deps = [
+        "inference_engine",
+        "mkldnn_plugin",
```

Comment:
I'm surprised this works, we should prefix with `:` (i.e. `":inference_engine"`)

Reply:
Indeed it does, but sure